src/main/java/frc/robot/RobotContainer.java

```diff
@@ -165,8 +165,22 @@ private void configureBindings() {
         .b()
         .onTrue(new InstantCommand(() -> scoreLoc.setScoreCol(ScoringLocationUtil.ScoreCol.RIGHT)));
 
-    operatorController.back().onTrue(new InstantCommand(intake::retract, intake));
-    operatorController.start().onTrue(new InstantCommand(intake::deploy, intake));
+    operatorController
+        .back()
+        .onTrue(
+            new InstantCommand(
+                () -> {
+                  intake.setDesiredDeployed(true);
+                },
+                intake));
+    operatorController
+        .start()
+        .onTrue(
+            new InstantCommand(
+                () -> {
+                  intake.setDesiredDeployed(false);
+                },
+                intake));
```

Comment:
I think these need to be flipped. back button for retract and start button for deploy, not vice versa